src/server/CMakeLists.txt

```diff
@@ -53,15 +56,75 @@ set(SOURCES attrdesc.cpp
         utils.cpp
         utils_polling.cpp
         utils_shut.cpp
+        w32win.cpp
         w_attribute.cpp
         w_pipe.cpp
         zmqeventsupplier.cpp)
-
+ELSE()
```

Comment:
Same as above - use lower case. Plus it is a good practice to put variable for long clauses:

```
else(WIN32)
```

See [documentation](https://cmake.org/cmake/help/v3.0/command/if.html)